R/driver-oracle.R

```diff
@@ -6,6 +6,13 @@ NULL
 #'
 #' Details of Oracle methods for odbc and DBI generics.
 #'
+# while other sections are documented with `@rdname Oracle` and a @description,
```

Comment:
Couldn't you instead remove `@param name` from `DBI-method`?

Reply:
Ah, I missed an `@inheritParams` that brought in that `name` documentation from DBI—I had incorrectly assumed this was some sort of S4 + roxygen2 automatic `@inheritParams` type thing.